src/components/MoviesList/MoviesList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './MoviesList.scss';
+import MovieCard from '../MovieCard/MovieCard';
+import movies from '../../api/movies.json';
+
+export const MoviesList = () => (
+  movies.map(({title, description, imgUrl, imdbUrl, imbdId}) => {
+   return (
+    <MovieCard
+        title={title}
+        description={description}
+        imgUrl={imgUrl}
+        imdbUrl={imdbUrl}
+        imbdId={imbdId}
+      />)
```

Comment:
If you map `movies`, key should also be written here.
```suggestion
  movies.map((movie) => {
   return (
    <MovieCard
      {...movie}
      key={movie.imdbId} 
     />)
```